### ISSUE TITLE
add `request-ci` label for PRs made modifying `tools/` in core repo

### ISSUE THREAD

**Issue by DerekNonGeneric** [nodejs/github-bot#308]
This was requested by @BridgeAR in https://github.com/nodejs/node/issues/41102, but had not yet been fully resolved by the PR that closed that issue. I have not contributed to this repo is a while, but may be able to handle it here if there is any interest in getting this taken care of here.

/cc @Trott @aduh95

**Comment by DerekNonGeneric** [comment]
I will have to put this off until a fully-green CI can be made achieveable even once.

The `master` branch appears to be currently making good progress towards that goal.

**Comment by phillipj** [comment]
Hi @DerekNonGeneric,

automagic PR labelling was indeed done by this project for years. That was changed earlier this year tho, by moving that responsibility into a GitHub Action instead: [nodejs/node-pr-labeler](https://github.com/nodejs/node-pr-labeler).

Main goal by doing so was to move the logic related to `file path -> label` _into_ the project using the GitHub Action, instead of inside the GitHub Action project (or github-bot). For nodejs/node that configuration lives in: [nodejs/node/blob/master/.github/label-pr-config.yml](https://github.com/nodejs/node/blob/master/.github/label-pr-config.yml).

**Bottom line:** I think what you're asking will be solved by changing the above configuration file inside nodejs/node.

Let me know if you need some help getting there :)
